airbyte-integrations/connectors/source-google-sheets/google_sheets_source/google_sheets_source.py

```diff
@@ -119,8 +123,7 @@ def read(
 
                 for row in row_values:
                     if Helpers.is_row_empty(row):
-                        encountered_blank_row = True
```

Comment:
I'm not sure about the correctness of these changes, but the previous code stopped reading data after it found one empty line, for example, if after the header line there is an empty line and then data, then we do not read this data. I made it so that all data in a specific ROW_BACTH_SIZE was read, and the cycle was interrupted when there were no records in this values_range.
Let me know please, if this is not the correct logic, I will revert the code to the previous state.